packagedoc/additional_docs/History.tex

```diff
@@ -96,6 +96,6 @@
 - Replaced \texttt{re} package by \texttt{regex} package.}
 
 \historyversiondate{0.8.1}{5/2025}
-\historychange{Updated dependency package.}
+\historychange{Module added to the package requirements.}
```

Comment:
Which one?

\texttt{regex} module added to the package requirements

Reply:
Hello Holger,

Thank you for the finding, I updated it.